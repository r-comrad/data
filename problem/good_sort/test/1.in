3
5231 1000 1002 