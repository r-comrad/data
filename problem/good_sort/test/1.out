1000 1002 5231 